components/layout/Sidebar/SubSidebar.tsx

```diff
@@ -242,6 +243,15 @@ const SubSidebar: React.FC<SubSidebarProps> = ({
             content="Events"
             isActive={pathname === `/spaces/${spaceId}/events`}
           />
+          {spaceId ==
+            'kjzl6kcym7w8y7drgmopt1aufcut7p9cbwyoaa0ht9vl8sgs5q39blhgsbeyb83' && (
```

Comment:
Why is this hardcoded?

Reply:
It should at least be a const, ok?

Reply:
hey yo @FrancoAguzzi ! I added the comment here about it: https://github.com/blockful-io/ZuzaluCityOS/pull/2#issue-2609742152

> For now we have the spaceId hardcoded to test the application. But will be changed to [spaceId enabled by trustful](https://github.com/blockful-io/ZuzaluCityOS/compare/feat/milestone-trustful-zuzalu-dApp-interface...heronlancellot:feat/welcome-text-role?expand=1#diff-f9ddf54b8b8f2d0e7b94f438dbcadf176d0f776cb94acb9b37f058c68abac7d2R21) ( CypherHouseSpaceId )

Reply:
and why does it need to wait?

Reply:
Because I only can test the application with this spaceIs hardcode. I created the space (Trustful Integration) and will only appear in Sidebar in this space. The another Space doesn't exist in development, it's a production spaceId who will be used in the Trustful integration. I can create a TODO in code, or change the codebase to the real one, but to test the application I need this space id for now, because of that I suggested to keep like that for now. After it, will be changed when we open the PR to ZuCity.

Reply:
@heronlancellot please:

1.  store this value inside a const that is clearly named
2. reuse this const here
3. add a comment above the const definition saying this is for temporary use
4. create one github issue for future const removal / replacement

this should give sufficient awareness

Reply:
@heronlancellot can you link the new task in here, please?

Reply:
https://github.com/blockful-io/frontend-zucity/issues/8 @FrancoAguzzi